### PR TITLE
consider case-sensitivity when logging in

### PR DESCRIPTION
**Pull request by mkxx712**
# Description
Add a small function to avoid failure when logging in.

## Related Issue
#671 

## Motivation and Context
With no idea that I need to change all my address character into all upper/lower case, I fail to log in.

## How Has This Been Tested
Use an address contains both upper and lower address and can log in successfully.

## Screenshots (if appropriate)
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

**Review by Kasshern — CHANGES_REQUESTED** [review]

File: examples/bri-3/src/bri/auth/agent/auth.agent.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class AuthAgent {
       return false;
     }
 
-    const isValid = publicKeyFromSignature === senderPublicKey;
+    const isValid = publicKeyFromSignature.toLowerCase() === senderPublicKey;
```
[the remark this answers is not in the document]
Comment (Manik-Jain) [reply examples/bri-3/src/bri/auth/agent/auth.agent.ts:47]: when web3 library reads current MM address, the default is lowercase.

**Review by Manik-Jain — DISMISSED** [review]

File: examples/bri-3/src/bri/auth/agent/auth.agent.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ognjenkurtic) [reply examples/bri-3/src/bri/auth/agent/auth.agent.ts:47]: +1 for lowercase on both sides. We are not using this code only for logging in, and public key is an input that is coming from the outiside so we do not have any control over its format

[the remark this answers is not in the document]
Comment (Kasshern) [reply examples/bri-3/src/bri/auth/agent/auth.agent.ts:47]: @mkxx712  Please fix and request approval again

[the remark this answers is not in the document]
Comment (mkxx712) [reply examples/bri-3/src/bri/auth/agent/auth.agent.ts:47]: Thanks for all the comments!

**Review by Kasshern — DISMISSED** [review]

File: examples/bri-3/src/bri/auth/agent/auth.agent.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ognjenkurtic) [reply examples/bri-3/src/bri/auth/agent/auth.agent.ts]: Please run prettier and we can merge

[the remark this answers is not in the document]
Comment (mkxx712) [reply examples/bri-3/src/bri/auth/agent/auth.agent.ts]: @ognjenkurtic Done (Sorry about that!

**Review by Manik-Jain — APPROVED** [review]

**Pull request merged**